cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceConfigMapUtilsTest.java

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.cluster.operator.assembly;
+
+import io.fabric8.kubernetes.api.model.ConfigMap;
+import io.fabric8.kubernetes.api.model.ConfigMapBuilder;
+import io.strimzi.api.kafka.model.rebalance.KafkaRebalanceState;
+import io.strimzi.operator.cluster.operator.resource.cruisecontrol.ExecutorStatus;
+import io.strimzi.operator.common.model.cruisecontrol.CruiseControlExecutorState;
+import org.junit.jupiter.api.Test;
+
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.Map;
+
+import static io.strimzi.operator.cluster.operator.assembly.KafkaRebalanceAssemblyOperator.BROKER_LOAD_KEY;
+import static io.strimzi.operator.cluster.operator.assembly.KafkaRebalanceConfigMapUtils.BYTE_MOVEMENT_COMPLETED;
+import static io.strimzi.operator.cluster.operator.assembly.KafkaRebalanceConfigMapUtils.COMPLETED_BYTE_MOVEMENT_PERCENTAGE_KEY;
+import static io.strimzi.operator.cluster.operator.assembly.KafkaRebalanceConfigMapUtils.ESTIMATED_TIME_TO_COMPLETION_IN_MINUTES_KEY;
+import static io.strimzi.operator.cluster.operator.assembly.KafkaRebalanceConfigMapUtils.EXECUTOR_STATE_KEY;
+import static io.strimzi.operator.cluster.operator.assembly.KafkaRebalanceConfigMapUtils.TIME_COMPLETED;
+import static io.strimzi.operator.cluster.operator.assembly.KafkaRebalanceConfigMapUtils.updateRebalanceConfigMapWithProgressFields;
+import static io.strimzi.operator.cluster.operator.assembly.KafkaRebalanceStatusTest.buildOptimizationProposal;
+import static io.strimzi.operator.cluster.operator.resource.cruisecontrol.ExecutorStatusTest.createExecutorStatusJson;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class KafkaRebalanceConfigMapUtilsTest {
+
+    private static final String BROKER_LOAD = buildOptimizationProposal().toString();
+    private static final Map<String, String> BROKER_LOAD_MAP = Map.of(BROKER_LOAD_KEY, buildOptimizationProposal().toString());
+
+    private static ConfigMap createKafkaRebalanceConfigMap(Map<String, String> brokerLoad) {
+        return new ConfigMapBuilder()
+                .addToData(brokerLoad)
+                .build();
+    }
+
+    @Test
+    public void testProgressFieldsForProposalReadyState() throws Exception {
```

Comment:
None of these methods actually `throws Exception`, so you can remove it

Reply:
Fixed.